src/steps/users/contact-create-or-update.ts

```diff
@@ -52,6 +74,16 @@ export class CreateOrUpdateContact extends BaseStep implements StepInterface {
     }
   }
 
+  public createRecord(contact, contactExist): StepRecord {
+    const obj = {};
+    Object.keys(contact).forEach((key: string) => {
+      if (isString(contact[key])) {
```

Comment:
- [ ] Do you know what values this is filtering out?  It's possible/likely that we'll want to return boolean and numeric values here too.
- [ ] I also noticed the `signupDate` and `profileUpdatedAt` values returned by Iterable appear to be _close-but-not-quite_ ISO 8601 (e.g. `2020-02-25 20:46:01 +00:00` vs `2020-02-25T20:46:01Z`.  We may want to coerce those values here.

Reply:
Sorry, was trying to filter out object values. Should've been `!isObject(contact[key]`. Should go with that or should object values be included? 

As for the second one i might have to use regex since `isDate(date)` is not working on their `date` values which are also returned as string

Reply:
👍